src/container/srv_container.c

```diff
@@ -3135,6 +3135,9 @@ set_prop(struct rdb_tx *tx, struct ds_pool *pool,
 	if (rc != 0)
 		goto out;
 
+	if (DAOS_FAIL_CHECK(DAOS_CHK_CONT_BAD_LABLE))
+		goto out;
```

Comment:
I don't think we need this either. We have `dmg faults mgmt-svc pool $pool POOL_BAD_LABEL` which will force the label in the MS to be different from the label stored as a property.

Reply:
This is for container label, not pool label.

Reply:
Ah, right. Of course... I didn't read it carefully.